molecule/default/requirements.yml

```diff
@@ -0,0 +1,4 @@
+---
+- src: https://github.com/cisagov/ansible-role-pip
+  name: pip
+  version: bugfix/add-kali-specific-vars
```

Comment:
Do we want to log an issue to remove `version: bugfix/add-kali-specific-vars` after https://github.com/cisagov/ansible-role-pip/pull/12 is merged?

Reply:
No, I don't need an issue for that.

Reply:
Famous last words...